CHANGELOG.md

```diff
@@ -57,9 +57,12 @@
 - Remove the requirement for INVALID_STATE for the cxx::expected [\#987](https://github.com/eclipse-iceoryx/iceoryx/issues/987)
 - Add unique test identifers [\#988](https://github.com/eclipse-iceoryx/iceoryx/issues/988)
 - Remove `ApplicationPort` and `ApplicationPortData` classes [\#415](https://github.com/eclipse-iceoryx/iceoryx/issues/415)
+- Updating Codecov API and enforce CMake version 3.16 for building iceoryx [\#774](https://github.com/eclipse-iceoryx/iceoryx/issues/774) and [\#1031](https://github.com/eclipse-iceoryx/iceoryx/issues/1031)
 
 **API Breaking Changes:**
 
+The CMake files in iceoryx expect to have CMake version 3.16 or greater installed, otherwise the build fails.
```

Comment:
Maybe add that for ubuntu 18.04 the `cmake-mozilla` package from the `universe` repo could be used

Reply:
Added this as a hint in Changelog